README.md

```diff
@@ -416,6 +416,81 @@ It works out of the box with:
 - A default TipTap editor configuration
 - Automatic rendering of registered Storyblok blocks using the `Storyblok\Bundle\Block\BlockRegistry`
 
+## Enabling Storyblok’s Live Editor
+
+This integration lets the Storyblok Visual Editor highlight components directly on your frontend and open the corresponding editing form automatically. Here’s how to set it up:
+
+1. **Load the Storyblok bridge script**
+
+In your `base.html.twig` layout, include the Storyblok JavaScript bridge — **but only in draft mode**:
+
+```diff
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Hello world</title>
+</head>
+<body>
+
+    {% block body %}{% endblock %}
+
++   {{ storyblok_js_bridge_scripts() }}
+</body>
+</html>
+```
+
+This script is responsible for detecting editable components on the page and opening the Live Editor when clicked.
+
+2. **Make your block classes “editable”**
+
+Every block you want editable in the Live Editor must implement the `EditableInterface` and use the `EditableTrait`,
+allowing them to receive Storyblok’s `_editable` metadata:
+
+```diff
+<?php
+
+declare(strict_types=1);
+
+namespace App\Bridge\Storyblok\Value\Block\Alert;
+
++ use Storyblok\Api\Domain\Type\Editable;
++ use Storyblok\Bundle\Editable\EditableInterface;
++ use Storyblok\Bundle\Editable\EditableTrait;
+use Storyblok\Bundle\Block\Attribute\AsBlock;
+
+#[AsBlock]
+-final readonly class MyBlock
```

Comment:
if we show a diff, I think there is no need to show alle use and declare strict types etc. its just noisy